docs/concepts/trading-framework/data-sources.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 8
+title: Data sources
+hide_title: false
+---
+Data sources, which come from oracles, are essential to markets created on Vega.
+
+Market settlement, risk models, and other features require a supplied price, or other data, which must come from somewhere, usually completely external to Vega. Specifically for settlement values, Vega requires external data.
+
+Vega APIs and protocol capabilities support a wide range of data sourcing styles and standards, and Vega will not integrate directly with specific oracles / data providers at the protocol level. This framework is flexible to make it easy to create and combine oracles, and therefore to design markets on almost anything.
```

Comment:
This is more aspirational than current 🥹

Reply:
Ah - yeah I can see that. I'll rework it to be clear what's true now and what's expected to be true  in the future :)